### PR TITLE
Just a little api update

### PR DESCRIPTION
**Pull request by da123rrell**

### CODE REVIEW

File: src/MSpawns/Commands/Hub.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+:<?php
```
Comment (TheDiamondYT1) [inline src/MSpawns/Commands/Hub.php]: Syntax error

File: src/MSpawns/Commands/SetAlias.php [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?php
+ommand<?php
```
Comment (TheDiamondYT1) [inline src/MSpawns/Commands/SetAlias.php]: Syntax error

**Comment by TheDiamondYT1** [comment]
You need to learn git as well 😀

**Comment by da123rrell** [comment]
Yeah, thanks :D

**Comment by TheDiamondYT1** [comment]
@da123rrell Read my comment

**Pull request closed without merging**